OxfordBodleian/Juel-Jensen/BDLaethf25.xml

```diff
@@ -153,21 +586,226 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        <q xml:lang="gez">
+                                            ዝሀወት<hi rend="rubric">፨</hi> ዘአበረ፡ አጋዘ<hi rend="rubric">፨</hi> 
+                                            ለአበወሀ<hi rend="rubric">፨</hi> ወ<hi rend="rubric">=</hi>ሰሐበ<hi rend="rubric">፨</hi> 
+                                            ሙሐዘት<hi rend="rubric">፨</hi> አየአላፈ<hi rend="rubric">፨</hi> 
+                                            ወፀበላነ። <add place="above">የመጦሪ<hi rend="rubric">፨</hi><hi rend="rubric">፨</hi></add>
+                                        </q>.
+                                </item>
+                                                                
+                                <item xml:id="a9">
+                                    <locus from="77v" to="78r"/>
+                                    <desc type="GuestText"><title type="complete" ref="LIT6862PPscribe"></title>
+                                        The text has been written on top of an erased unidentified text written in pencil.
+                                    </desc>
+                                        <q xml:lang="gez">
+                                            <hi rend="rubric">
+                                                ሰሐል፡ ሰሐል፡ ሰሐል፡ ጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸ<del rend="strikethrough">ጸ</del>እዴየ፡ 
+                                                ሰሐል፡ ወየማንየ፡ አሕመላምል፡ ለጺሒፈ፡ ብርዕ፡ ወቀለም፡ ሊተ፡ ለገብርከ፡ አፍጌት፡ 
+                                                ወርኃ፡ ደቡብ፡ ወወርኃ፡ ደመና፡ <del rend="encircled strikethrough">አክሄል፡ ድሜሄላዊ፡</del> 
+                                                መንጥሔል፡ ድሜሄላዊ፡ በከመ፡ የሐውሩ፡ ፀሐይ፡ ወወርኅ፡ በፍጥረተ፡ 
+                                                ትእዛዝከ፡ ከማሆሙ፡ አሩፅ፡ የማነ፡ እዴየ፡ ለጋ፡ በስመ፡ 
+                                                <del rend="strikethrough">ለለ</del>ሎኤ፡ ኢላኤ፡ ኢላሄ፡ አሩፅ፡ እዴየ፡ 
+                                                ለገቢ<pb n="78r"/>ረ፡ ኵሉ፡ ተግባር፡ በሩፋኤል፡ ብርናዊ፡ 
+                                                በክሳብኤል፡ ዘ<del rend="strikethrough">ፈ</del>ያረውፃ፡ ለፀሐይ፡ 
+                                                ከማሁ፡ አሩፅ፡ እዴየ፡ ከመ፡ ይጠንቅቅ፡ ለአኂዘ፡ ብርዕ፡ ወእጽሐፍ፡ 
+                                                እበያቲከ፡ ዘእንበለ፡ ሐኬት፡ ኦእግዚእየ፡ ኢየሱስ፡ ክርስቶስ፡ 
+                                                አስተ<surplus>፡</surplus> ፋጥን<supplied reason="undefined">፡</supplied> 
+                                                ወ<add place="above">አ</add>ስተ<surplus>፡</surplus> 
+                                                ቃንዕ፡ ለገብርከ፡ በቀይ፡ ቀለም፡ <del rend="strikethrough">ወ</del>በወረቀት፡ 
+                                                ጸሐፍዕስር<supplied reason="undefined">፡</supplied>
+                                            </hi>
+                                        </q>.
+                                </item>
+
+                                <item xml:id="e1">
+                                    <locus target="#1r"/>
+                                    <desc type="OwnershipNote"> 
```

Comment:
e1 does not need _type_ inside `<desc/>`. I would describe this note as such: "The current signature and the title of the first work have been added in pencil on the upper outer corner of the page".
Also, I read "Widassie Maryam".
Please do not forget the white stickers glued onto the inner side of the front board, which contain the name of `<persName ref="PRS5782JuelJen"/>` written in fidal (<foreign xml:lang="gez">ቤንት፡ ዩውል፡ የንሰን።</foreign>) and the signature 'MS. Aeth. f. 25' in pencil. The second sticker also contain another signature ‘Ms 27’.